demo/js/aggregations-view.js

```diff
@@ -32,7 +32,7 @@ window.onload = function() {
             [Hypergrid.AggregationsView, options] // object API instantiation with one arg; `$$CLASS_NAME` defined so ref saved in `grid.plugins.aggregationsView`
         ]
     });
-
+    // These modules are for EXAMPLE purposes only
```

Comment:
but it's in the build. i would hesitate to add these comments until we actually remove it from the build.